docs/architecture-decisions/0005-supported-browsers.md

```diff
@@ -0,0 +1,53 @@
+# 5. Supported Browsers
+
+Date: 2020-11-06
+
+## Status
+
+Accepted
+
+## Context
+
+Writing applications to support every possible web browser is, at best, tedious
+and error prone. New technologies are interesting to work with, but may not be
+available on older browsers. We don't want to insist our users always have the
+latest version of technology to access our resources, but we also can't support
+browsers so old they hamper development and are not even supported by the
+browser developers.
+
+## Decision
+
+We should support the browsers that our users regularly use on our site.
+
+Rather than defining a specific list of browsers which will likely be out of
+date before the next time someone needs it, we will instead make it our goal to
+ensure our applications are optimized for the browsers in 99% of our sessions,
+do a best effort for the next 0.5% (it should work, but may not be
+a great user experience), and allow the last 0.5% to work or not as may be with
```

Comment:
From EngX discussion: should we only use thresholds from non-bounced traffic or all traffic?